src/main/java/com/puppycrawl/tools/checkstyle/checks/NewlineAtEndOfFileCheck.java

```diff
@@ -88,6 +88,19 @@
  * <pre>
  * &lt;module name=&quot;NewlineAtEndOfFile&quot;/&gt;
  * </pre>
+ * <p>Example:</p>
+ * <pre>
+ * // File ending with a new line
+ * public class Test {
+ *
+ * }
```

Comment:
Please add symbol ⤶ to the end of the each line (\u2936) to make it clear.

Reply:
Added 👍